apps/files/src/components/FileList.vue

```diff
@@ -2,51 +2,29 @@
   <div class="uk-height-1-1">
     <div class="uk-flex uk-flex-column uk-height-1-1">
       <div id="files-list-container" class="uk-overflow-auto uk-flex-auto">
-        <oc-table middle divider class="oc-filelist uk-margin-remove-bottom" id="files-list" v-show="!loadingFolder">
+        <oc-table middle divider class="oc-filelist uk-margin-remove-bottom" :id="id" v-show="!loadingFolder">
```

Comment:
Is the condition for the displaying of the list good here? Since we already pass it as a prop, wouldn't be better to do it directly where the component is imported?

Reply:
Ah, my bad. Thought it's a prop. Now I see it's a state. Sorry